sss/process_foms.c

```diff
@@ -365,6 +379,36 @@ static int ss_process_stats(struct m0_reqh           *reqh,
 		rc = ss_ios_stats_ingest(rep);
 	return M0_RC(rc);
 }
+
+static int ss_bytecount_stats_ingest(struct m0_ss_process_rep *rep)
+{
+	struct m0_cob_bckey  key;
+	struct m0_cob_bcrec  rec;
+
+	/* DUMMY values */
+	key.cbk_pfid = M0_FID_TINIT('v', 1, 8);
+	key.cbk_user_id = 8881212;
+
+	rec.cbr_bytecount = 10240000;
+	rec.cbr_cob_objects = 10000;
+
```

Comment:
Make sure ioservice cob domain is accessible here.

Reply:
Verified cob domain to be accessible here.